Makefile

```diff
@@ -1,6 +1,18 @@
+DOCKER_SERVICES ?= all
+
 help:
 	@grep '^[^#[:space:]].*:' Makefile
 
+# Usage:
+# `make-docker-compose-up` starts all the services.
```

Comment:
`make-docker-compose-up` or `make docker-compose-up` ?

not sure if it's a typo or a convention in a makefile.

Reply:
A typo, thanks for pointing that out.